AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -1974,7 +2054,9 @@ - (void)layout
   ASDisplayNode *subnode = nil;
   CGRect subnodeFrame = CGRectZero;
   for (ASLayout *subnodeLayout in _layout.sublayouts) {
-    ASDisplayNodeAssert([_subnodes containsObject:subnodeLayout.layoutableObject], @"Cached sublayouts must only contain subnodes' layout.  self = %@, subnodes = %@", self, _subnodes);
+    if (![[self class] usesImplicitHierarchyManagement]) {
+      ASDisplayNodeAssert([_subnodes containsObject:subnodeLayout.layoutableObject], @"Cached sublayouts must only contain subnodes' layout.  self = %@, subnodes = %@", self, _subnodes);
```

Comment:
Maybe delete the word Cached from the log message at the beginning, not sure it makes sense